rules-xml/impl/src/main/java/org/jboss/windup/rules/apps/xml/xml/ProcessingIsOnlineGraphCondition.java

```diff
@@ -0,0 +1,20 @@
+package org.jboss.windup.rules.apps.xml.xml;
+
+import org.jboss.windup.config.GraphRewrite;
+import org.jboss.windup.config.condition.GraphCondition;
+import org.jboss.windup.graph.model.WindupConfigurationModel;
+import org.jboss.windup.graph.service.GraphService;
+import org.ocpsoft.rewrite.context.EvaluationContext;
+
+/**
+ * A {@link GraphCondition} returning true/false based on the fact if the current processing is online/offline.
+ */
+public class ProcessingIsOnlineGraphCondition extends GraphCondition
```

Comment:
Good idea... I really thought we already had this. :)

I would suggest moving it to rules-base, though.